desktop/window-handlers/external-links/index.js

```diff
@@ -45,8 +49,16 @@ function isValidBrowserUrl( url ) {
 	return false;
 }
 
+function isWpAdminLogin( url ) {
```

Comment:
Ideally, we should be able to get a site's definitive Jetpack designation, but I'm unsure how to do this. This is the best proxy I could come up with by inspecting the Jetpack redirect URLs.

If you use the WP.com API to query a site with `GET https://public-api.wordpress.com/rest/v1.1/<domain>`, one of the elements returned in the JSON response is a `jetpack` boolean. This is even better, but we would have to insert a blocking HTTP request to fetch this info - not sure if that's a good strategy, but is certainly an option.

It's possible we might get this from Calypso for free somehow, but am unsure where to look/how to hook into this info.

Reply:
this should be available in the [`getSelectedSite()`](https://github.com/Automattic/wp-calypso/blob/master/client/state/ui/selectors/get-selected-site.js) selector. You'll need to get it from the Redux store but I think you can do that with a single import - it could be more than this but regardless it's possible.

after that you should even have access to [`getJetpackSetting()`](https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/get-jetpack-setting.js) or whatever it is that will tell us if we have SSO enabled.